### ISSUE TITLE
how to train your model

### ISSUE THREAD

**Issue by gaopeng-eugene** [zck119/3dgan-release#1]
Will you release your training code?

**Comment by meetps** [comment]
@gaopeng-eugene @zck119  I've implemented the exact same model as mentioned in the paper in Tensorflow [here](https://github.com/meetshah1995/tf-3dgan). But I'm unable to reproduce the results after a lot of failed attempts. I too would appreciate the release of the training code.

**Comment by Jerrypiglet** [comment]
Same situation here. Can you please release the training code? Thanks!

**Comment by xhygh** [comment]
How to convert from .off or .obj files to .mat files?

**Comment by KnightOfTheMoonlight** [comment]
+1

**Comment by raahii** [comment]
Only for voxel generation, I got reasonably result on my [chainer implemention](https://github.com/piyo56/3dgan-chainer), but successful conditions of model structure or hyper-parameters  are not same as the thesis.
I also want you to release the training code. Thanks.

**Comment by EdwardSmith1884** [comment]
If you guys want a way to train their model, I have a working tensor flow implementation on [here](https://github.com/EdwardSmith1884/3D-IWGAN/tree/master/3D-Generation). It's at 32 by 32 by 32 resolution instead, but it trains very quickly. I also have a implementation which uses wasserstein loss, which doesn't need parameter tuning to train, and should produce high quality models.
